src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,25 @@
+const STATUS = 'Under construction';
+
+function categoriStatus() {
+  return {
+    type: STATUS,
+  };
+}
+
+const initialState = {
+  catagories: [],
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case STATUS:
+      return {
+        state,
+      };
+    default:
+      return state;
+  }
+};
+
+categoriStatus();
+reducer();
```

Comment:
Just as I requested in the previous review, Kindly use the createSlice method from @reduxjs/toolkit to add your reducers so as to have a single or combined reducer that can be exported👍. This function helps to reduce the boilerplate required to add data to redux in a canonical way😉

you can use this method by importing it from @reduxjs/toolkit like this import { createSlice } from "@reduxjs/toolkit" and kindly export the various actions and your reducer👍.